core/go.mod

```diff
@@ -0,0 +1,3 @@
+module github.com/redhat-developer/app-services-sdk-go/core
+
+go 1.13
```

Comment:
"core" does not need to be its own separate module - it is part of the root module. You can just remove this file and leave everything else as is.